### ISSUE TITLE
Case conflict for Polygon.js

### ISSUE THREAD

**Issue by Bjvanminnen** [d3/d3-delaunay#39]
I get the following error when I try to use this library on Linux:
```
Module not found: Can't resolve './polygon' in '/home/brent/git/terrain3/node_modules/d3-delaunay/src'
```
This is because the file is named Polygon.js, but gets imported as "polygon".
https://github.com/d3/d3-delaunay/blob/master/src/delaunay.js#L3

Renaming the file to "polygon.js" in my node_modules fixes the issue.

**Comment by mbostock** [comment]
Argh, the case-insensitive file system of macOS. Fix to be released shortly…

**Comment by mbostock** [comment]
Fixed in 3.1.1.
